res/loc/po/zh-CN.po

```diff
@@ -288,39 +288,39 @@ msgstr "字节"
 #.
 #. *Short* version of the kilobyte size suffix
 msgid "KB"
-msgstr ""
+msgstr "KB"
 
 #. • MSG_022
 #.
 #. *Short* version of the megabyte size suffix
 msgid "MB"
-msgstr ""
+msgstr "MB"
 
 #. • MSG_023
 #.
 #. *Short* version of the gigabyte size suffix
 msgid "GB"
-msgstr ""
+msgstr "GB"
 
 #. • MSG_024
 #.
 #. *Short* version of the terabyte size suffix
 msgid "TB"
-msgstr ""
+msgstr "TB"
 
 #. • MSG_025
 #.
 #. *Short* version of the pentabyte size suffix
 msgid "PB"
-msgstr ""
+msgstr "PB"
 
 #. • MSG_027
 msgid "kilobytes"
-msgstr "K 字节"
+msgstr "k 字节"
 
 #. • MSG_028
 msgid "megabytes"
-msgstr "M 字节"
+msgstr "兆 字节"
```

Comment:
I'm not sure I understand why kilobytes became "k 字节" but megabytes becomes "兆 字节". The rule of whether an English letter should be used before "字节" doesn't seem that logical. Also I would expect that a capital letter would be used, since it derives from English and a capital letter is always being used for KB or MB. All in all, I'm not sure why this change is needed.